src/main/java/com/atlassian/jira/cloud/jenkins/config/JiraCloudPluginConfig.java

```diff
@@ -98,13 +102,18 @@ public boolean configure(final StaplerRequest req, final JSONObject json) throws
                 }
             }
 
+            if (json.containsKey(FIELD_NAME_DEBUG_LOGGING)) {
+                this.debugLogging = json.getBoolean(FIELD_NAME_DEBUG_LOGGING);
+            }
+
         } catch (Exception e) {
-            log.debug("Submitting form to JSW Plugin failed: ({})", e.getMessage(), e);
+            log.debug("Submitting form to Atlassian Cloud plugin failed: ({})", e.getMessage(), e);
             if (log.isTraceEnabled()) {
-                log.trace("JSW form data: {}", json.toString());
+                log.trace("Atlassian Cloud plugin data: {}", json.toString());
```

Comment:
Ultimately, the plugin should be named "Atlassian Cloud Plugin", because we might add stuff for JSM, as well.